CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ project(viam-cpp-sdk
   VERSION 0.0.0
   DESCRIPTION "Viam Robotics C++ SDK"
   HOMEPAGE_URL https://github.com/viamrobotics/viam-cpp-sdk
-  LANGUAGES CXX
+  LANGUAGES C CXX
```

Comment:
I think this can be backed out. I'd needed it at one point while I was experimenting with different ways to make a soft dependency on tflite, but I don't think the mechanism I settled on requires it.